test/support/graphql/mutations_variable.rb

```diff
@@ -35,6 +35,27 @@ def add_wallet_mutation_variables(attrs = {})
       camelize_hash_keys(attrs).to_json
     end
 
+    def add_category_group_mutation_variables(attrs = {})
+      category_group_attrs = attributes_for(:category_group)
+      category_group_attrs[:title] = 'category group'
```

Comment:
you can define this in the instead, it's easier to track the expected result

Reply:
something like this https://github.com/jaimejazarenoiii/kwartracker-api/pull/7/files#diff-186c15ae5f227dea1cc41d7ea7ef85f5f775c77127bc90044a52a9cbcf7f637cR14